packit/cli/propose_downstream.py

```diff
@@ -40,64 +41,115 @@ def get_dg_branches(api, dist_git_branch):
     return get_branches(*dg_branches, default_dg_branch=default_dg_branch)
 
 
+def sync_release(
+    config,
+    dist_git_path,
+    dist_git_branch,
+    force_new_sources,
+    pr,
+    local_content,
+    path_or_url,
+    upstream_ref,
+    version,
+    force,
+    use_downstream_specfile,
+    package_config,
+):
+    api = get_packit_api(
+        config=config,
+        package_config=package_config,
+        dist_git_path=dist_git_path,
+        local_project=path_or_url,
+    )
+    if pr is None:
+        pr = api.package_config.create_pr
+
+    branches_to_update = get_dg_branches(api, dist_git_branch)
+
+    click.echo(
+        f"Proposing update of the following branches: {', '.join(branches_to_update)}"
+    )
+
+    for branch in branches_to_update:
+        api.sync_release(
+            dist_git_branch=branch,
+            use_local_content=local_content,
+            version=version,
+            force_new_sources=force_new_sources,
+            upstream_ref=upstream_ref,
+            create_pr=pr,
+            force=force,
+            use_downstream_specfile=use_downstream_specfile,
+        )
+
+
+def common_options(func):
```

Comment:
I would use something explicit.

```suggestion
def sync_release_common_options(func):
```